migrations/educandu-2023-11-20-02-add-midi-sound-to-ear-training-plugin.js

```diff
@@ -0,0 +1,60 @@
+export default class Educandu_2023_11_20_02_add_midi_sound_to_ear_training_plugin {
+  constructor(db) {
+    this.db = db;
+  }
+
+  async collectionUp(collectionName) {
+    const pluginType = 'ear-training';
+    const cursor = this.db.collection(collectionName).find({ 'sections.type': pluginType });
+
+    let count = 0;
+    while (await cursor.hasNext()) {
+      const doc = await cursor.next();
+
+      for (const section of doc.sections) {
+        if (section.type === pluginType && !!section.content) {
+          for (const test of section.content.tests) {
+            test.testMode = test.mode;
+
+            if (test.testMode === 'abcCode') {
```

Comment:
I'm not sure I get why this `if`-`else` is needed. Did you discover some data inconsistencies?

Reply:
I have not checked, but it is possible, because before we had the following validations:

~~~
questionImage: joi.alternatives().try(
  joi.object({
    sourceUrl: joi.string().allow('').required(),
    copyrightNotice: joi.string().allow('').required()
  }),
  joi.any().valid(null)
).required()
~~~

I removed the alternative in this PR making `null` invalid, so, just to be sure, I set the default empty values here.